x-pack/platform/plugins/shared/onechat/server/types.ts

```diff
@@ -84,4 +85,16 @@ export interface OnechatPluginStart {
    * Tools service, to manage or execute tools.
    */
   tools: ToolsStart;
+  /**
+   * Agents service, to execute agents.
+   */
+  agents: {
+    execute: RunAgentFn;
+  };
+  /**
+   * Chat service, to converse with agents with streaming support.
+   */
+  chat: {
+    converse: ChatService['converse'];
```

Comment:
Why do we need this chat.converse to be exposed?

Reply:
This is the only method that supports streaming. I didn't realize it created a conversation in agent builder... that's a problem